.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,26 +2,33 @@ Closes #
 
 {{short description}}
 
-#### Changelog
+### Changelog
 
 **New**
 
 - {{new thing}}
 
 **Changed**
 
-- {{change thing}}
+- {{changed thing}}
 
 **Removed**
 
 - {{removed thing}}
 
 #### Testing / Reviewing
 
-{{ Add descriptions, steps or a checklist for how reviewers can verify this PR works or not }}
+{{ Add steps or a checklist for how reviewers can verify this PR works or not }}
 
-<!--
-❗ Make sure you've included everything from the PR guide:
+## Merge Checklist
 
-https://github.com/carbon-design-system/carbon/blob/main/docs/guides/reviewing-pull-requests.md
--->
+As the author of this PR, confirm you:
+
+- [ ] Reviewed every line of the diff
+- [ ] Updated documentation and storybook examples
+- [ ] Wrote passing tests that cover this change
+- [ ] Addressed impacts to accessibility
+- [ ] Tested for cross-browser consistency
+- [ ] Validated that this code is ready for review and status checks should pass
```

Comment:
```suggestion
- [ ] Tested for cross-browser consistency
- [ ] Percy snapshots accurately reflect any visual changes made
- [ ] Validated that this code is ready for review and status checks should pass
```

Just a suggestion, but should we also remind to check Percy to make sure the changes are correct and there are no side effects for other components?

Reply:
Not a bad idea, though would a PR author be able to do that before opening the PR? 

Despite it being called "merge checklist" I guess I was hoping for it to be more of a "submission" checklist where everything could be done before opening the PR. The ending phrase "status checks should pass" being an exception of course.

Reply:
Makes sense!